index.html

```diff
@@ -272,5 +272,23 @@ <h3 class="sub_title">
         </ul>
       </section>
     </main>
+    <!-- contact form section  -->
+
+    <section class="contact-form">
+     <div class="container">
+      <div class="contact-text">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, 
+          a feature that you need built or a project that needs coding.
+           I’d love to help with it </p>
+       </div>
+       <form class="forms">    
+          <input type="text" class="name" placeholder="Enter your name" maxlength="30" required="">       
```

Comment:
I suggest you remove the `=""` on the required attribute. It does not require quotation marks. You can read about it [here](https://www.w3schools.com/tags/att_input_required.asp)